pkg/process/logging.go

```diff
@@ -81,3 +92,50 @@ func newLogger() (*zap.Logger, error) {
 		ErrorOutputPaths: []string{*logOutput},
 	}.Build()
 }
+
+type prettyEncoder struct {
+	*zapcore.MapObjectEncoder
+	config zapcore.EncoderConfig
+	pool   buffer.Pool
+}
+
+func newPrettyEncoder(config zapcore.EncoderConfig) *prettyEncoder {
+	return &prettyEncoder{
+		MapObjectEncoder: zapcore.NewMapObjectEncoder(),
+		config:           config,
+		pool:             buffer.NewPool(),
+	}
+}
+
+func (p *prettyEncoder) Clone() zapcore.Encoder {
+	rv := newPrettyEncoder(p.config)
+	for key, val := range p.MapObjectEncoder.Fields {
+		rv.MapObjectEncoder.Fields[key] = val
+	}
+	return rv
+}
+
+func (p *prettyEncoder) EncodeEntry(entry zapcore.Entry, fields []zapcore.Field) (*buffer.Buffer, error) {
+	b := p.pool.Get()
+
+	fmt.Fprintf(b, "%s\t%s\t%s\n",
+		entry.Time.Format("15:04:05.000"),
```

Comment:
The reason I included time-zone because it will be easier to correlate events from one log to the satellite log.

Reply:
I second that unless we force UTC time to be logged.

Reply:
Strong +1 from me (for the `"-0700"` spelling; the `"MST"` spelling is much less useful when logs might come from any tz on the planet).

In a similar vein, this needs to include the date. Logs will (hopefully) usually span multiple days, and for some log levels there will be gaps where nothing is logged for more than 24 hours.

Finally, would be good for this to respect `$STORJ_LOG_NOTIME` still